runner/runner/main.py

```diff
@@ -252,85 +260,104 @@ def process_plugin(plugin_config: Mapping[str, Any]) -> Path:
         env_override=env_monado,
     )
 
-    if not "openxr_app" in config["action"]:
-        raise RuntimeError(f"Missing 'openxr_app' property for action '{action_name}")
-
-    openxr_app_obj    : Mapping[str, Any] = config["action"]["openxr_app"]
-    openxr_app_config : Mapping[str, str] = openxr_app_obj.get("config", {})
-
-    openxr_app_path     : Optional[Path] # Forward declare type
-    openxr_app_bin_path : Path           # Forward declare type
-
-    if "src_path" in openxr_app_obj["app"]:
-        ## Pathify 'src_path' for compilation
-        openxr_app_path     = pathify(openxr_app_obj["app"]["src_path"], root_dir, cache_path, True , True)
-        openxr_app_bin_path = openxr_app_path / openxr_app_obj["app"]["bin_subpath"]
-    else:
-        ## Get the full path to the 'app' binary
-        openxr_app_path     = None
-        openxr_app_bin_path = pathify(openxr_app_obj["app"], root_dir, cache_path, True, False)
-
-    ## Compile the OpenXR app if we received an 'app' with 'src_path'
-    if openxr_app_path:
-        cmake(
-            openxr_app_path,
-            openxr_app_path / "build",
-            dict(CMAKE_BUILD_TYPE=cmake_profile, **openxr_app_config),
+    ## Compile OpenXR apps
```

Comment:
You're planning to compile all the OpenXR apps?
Does this also include the 2 Unreal apps that we have

Reply:
I believe this is addressed in the comment below.